packages/account/src/providers/transaction-response/transaction-response.ts

```diff
@@ -228,6 +228,13 @@ export class TransactionResponse {
 
     for await (const { statusChange } of subscription) {
       if (statusChange.type === 'SqueezedOutStatus') {
+        if (/BlobIdAlreadyUploaded/i.exec(statusChange.reason)) {
```

Comment:
Favour `indexOf` over regex for performance.

```suggestion
        if (statusChange.reason.toLowerCase().indexOf('blobidalreadyuploaded') > -1) {
```

or (if it doesn't need to case insensitive)

```suggestion
        if (statusChange.reason.indexOf('BlobIdAlreadyUploaded') > -1) {
```